html4j/src/main/java/org/teavm/html4j/HTML4jResourceSupplier.java

```diff
@@ -27,17 +26,10 @@
 public class HTML4jResourceSupplier implements ResourceSupplier {
     @Override
     public String[] supplyResources(ClassLoader classLoader, ListableClassReaderSource classSource) {
-        Set<String> resources = new HashSet<>();
-        for (String className : classSource.getClassNames()) {
-            final int lastDot = className.lastIndexOf('.');
-            if (lastDot == -1) {
-                continue;
-            }
-            String packageName = className.substring(0, lastDot);
-            String resourceName = packageName.replace('.', '/') + "/" + "jvm.txt";
```

Comment:
The jvm.txt was just an example, it doesn't have any special meaning in the HTML/Java API neither in the Controls4j applications.